app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class TestController extends Controller
+{
+    public function verificarRuta()
+    {
+        $response = Http::get('http://localhost/test'); // Cambia la URL según tu entorno
```

Comment:
You should always use .env to handle this type of [configuration](https://laravel.com/docs/10.x/configuration).
The controller is not a place to handle [testings](https://laravel.com/docs/10.x/testing).